### PR TITLE
PFT - latest changes/datatable update

### PR DESCRIPTION
**Pull request by prycrane**
Update to the latest changes page and the datatables include file.

**Preview**

- https://deploy-preview-348--design-system-canada-ca.netlify.app/about/latest-changes
- https://deploy-preview-348--design-system-canada-ca.netlify.app/
- https://deploy-preview-348--design-system-canada-ca.netlify.app/pattern-library

**French PR**

- https://github.com/canada-ca/systeme-conception/pull/228

### CODE REVIEW

File: about/latest-changes.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 altLangPage: "https://conception.canada.ca/a-propos/derniers-changements.html"
 date: 2017-10-05
-dateModified: 2023-03-28
+dateModified: 2023-10-20
```
Comment (donohuech) [inline about/latest-changes.md]: Please change to 2023-11-07

File: about/latest-changes.md [hunk 2]
```diff
@@ -14,6 +14,10 @@ title: "Latest changes to the Canada.ca design system"
 </div>
 <div id="expand-collapse">
   <details open="open">
+    <summary>October  20, 2023</summary>
```
Comment (donohuech) [inline about/latest-changes.md]: Please change to November 7, 2023

**Review by donohuech — CHANGES_REQUESTED** [review]
A few date changes please @prycrane

File: about/latest-changes.md (hunk 1 above)
Comment (prycrane) [reply about/latest-changes.md]: Done

File: about/latest-changes.md (hunk 2 above)
Comment (prycrane) [reply about/latest-changes.md]: Done

**Review by donohuech — APPROVED** [review]

**Pull request merged**